### ISSUE TITLE
PG Backups as an add-on has been deprecated.

### ISSUE THREAD

**Issue by franciscomxs** [Helabs/pah#473]
Pah fails to create new projects if you choose to create heroku apps, because `pgbackups` has been deprecated.

### Output:
```
Adding pgbackups:auto-month on app... failed
 !    Add-on plan not found.
```

### Source:
https://github.com/Helabs/pah/blob/master/lib/pah/templates/heroku.rb#L2

### Info:
https://devcenter.heroku.com/articles/pgbackups

```
PG Backups as an add-on has been deprecated. The commands exist as part of the
Heroku Postgres namespace in the CLI. The new functionality is live and available for
use. We also have a mapping guide to show you how the old commands transfer to
the new ones.
```

**Comment by TiuTalk** [comment]
@franciscomxs can you provide a PR?

Related to #471 and https://github.com/Helabs/jumpup-heroku/issues/72

**Comment by lunks** [comment]
Basically what we have to do is remove the addon and run `heroku pg:backups schedule DATABASE_URL --app sushi` on the heroku section.

**Comment by marcioj** [comment]
Hi guys thanks for track it down. Mind in give a look in #474?
I removed the inclusion of the addon and scheduled the backup using the @lunks command

**Comment by marcioj** [comment]
Merged in #474 and released a new version in 89883a0849ad49b206782eac70bbfb5e02318d08.
